admin/index.php

```diff
@@ -155,56 +183,89 @@
 			window.history.replaceState(null, null, window.location.href);
 		}
 	</script>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">	
 </head>
 
 <body>
-	<header>
-		<div class="logo cell">
-			<img class="responsive" src="../assets/img/outpost-icc-pm.png" alt="logo" title="ICC logo" />
+		<header>
+		<div class="header">
+			<img class="logo" src="../assets/img/outpost-icc-pm.png" alt="logo" title="ICC logo" /><br>
+				<div class="slider-section">
+					<form method="get">
+						<label class="switch" onchange="this.form.submit()">
+							<input type="checkbox" name="current_event" value="Yes" <?php if ($current_event == "Yes")
+								echo 'checked'; ?>>
+							<span class="slider round"></span>
+							<input type="hidden" name="tab" value="<?php echo $tab; ?>" />
+							<input type="hidden" name="faction" value="<?php echo $faction; ?>" />
+						</label>
+						<h4>
+							<?php
+							if (isset($_GET['current_event']) && $_GET['current_event'] == "Yes") {
+								echo 'This Event Only';
+							} else {
+								echo 'All active characters';
+							}
+							?>
+						</h4>
```

Comment:
You can't just set an h4 level out of nowhere. This is not a11y friendly. They need to be nested

Reply:
@rolf-yoast  - What do you mean by nested? Are you saying that an H4 needs to be inside an H3, inside an H2, inside an H1?